src/docmap-parser.ts

```diff
@@ -88,15 +89,19 @@ const getPreprintFromExpression = (expression: Expression): Preprint => {
     throw Error('Cannot identify Expression by DOI');
   }
   const content = [];
-  if (expression.url) {
-    content.push({ type: ContentType.Article, url: expression.url });
+  if (Array.isArray(expression.content) && expression.content.length > 0) {
+    content.push(...expression.content.map((contentItem) => contentItem.url).filter((url): url is string => !!url));
   }
+
+  const url = expression.url ? { url: expression.url } : {};
+
   return {
     id: expression.identifier ?? expression.doi,
     doi: expression.doi,
-    content: expression.url,
+    content,
     publishedDate: expression.published,
     versionIdentifier: expression.versionIdentifier,
+    ...url,
```

Comment:
Ditto to previous PR about this being `url: expression.url` and removing line 96